webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4196,6 +4198,14 @@ var MCK_BOT_MESSAGE_QUEUE = [];
                 });
             };
 
+            _this.adjustConversationTitleHeadingWidth = function() {
+                var titleClassName = 'mck-title-width-wo-faq-with-close-btn';
+                if(document.querySelector(".km-kb-container").classList.contains("vis")) {
```

Comment:
@CaptainT33mo can we add this check also  **document.querySelector(".km-kb-container")**